generators/python-v2/ast/src/TypeInstantiation.ts

```diff
@@ -145,6 +154,11 @@ export class TypeInstantiation extends AstNode {
             case "str":
                 writer.write(`"${this.internalType.value}"`);
                 break;
+            case "block_str": {
+                const suffix = this.internalType.value.endsWith("\n") ? "\\" : "\n";
+                writer.write(`"""\\\n${this.internalType.value}${suffix}"""`);
```

Comment:
This behavior would be handled in the `str` case in `if (this.internalType.block) { ... }`.